test/tests/functional/pbs_job_sort_formula.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2019 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestJobSortFormula(TestFunctional):
+    """
+    Tests for the job_sort_formula
+    """
+
+    def test_job_sort_formula_negative_value(self):
+        """
+        Test to see that negative values in the job_sort_formula
+        correctly sort properly
+        """
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.server.shortname)
+        self.server.manager(MGR_CMD_CREATE, RSC, {'type': 'float'}, id='foo')
+
+        a = {'job_sort_formula': 'foo', 'scheduling': 'False'}
+        self.server.manager(MGR_CMD_SET, SERVER, a, runas=ROOT_USER)
+
+        j1 = Job(TEST_USER, attrs={'Resource_List.foo': -2})
+        jid1 = self.server.submit(j1)
+        j2 = Job(TEST_USER, attrs={'Resource_List.foo': 1})
```

Comment:
-1 right?

Reply:
I meant 1 to be higher than -2, but I see your point.  I'll change it to -1.  The issue where the job was being ignored really had to do with -2, but testing them both is better.